include/mgmt/mgmt_fmfu_stat.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/** @file mgmt_fmfu_stat.h
+ * @defgroup mgmt_fmfu_stat Management Full Modem Firmware Upgrade Stats
+ * @{
+ * @brief Full Modem Firmware Update(FMFU) statistics.
```

Comment:
Again state already here that it is SMP based (or some other term to distinguish it from the OTA variant)